GitUI/FormSignatureInfo.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Data;
+using System.Drawing;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using System.Windows.Forms;
+
+namespace GitUI
+{
+    public partial class FormSignatureInfo : Form
+    {
+        public FormSignatureInfo()
+        {
+            InitializeComponent();
+        }
+
+        public void ShowGpgMessage(string message)
+        {
+            gpgTextBox.Text = message;
```

Comment:
The message needs new lines correction. `EnvUtils.ReplaceLinuxNewLinesDependingOnPlatform`
![gitext](https://user-images.githubusercontent.com/1003909/30695207-a08677e8-9ed7-11e7-93f8-979ca7cc8ba7.png)

Reply:
it's strange, on my windows 7 it has newlines, look at the images in this discussion.
I'll add this code.

Reply:
I tested it on `git://github.com/gitster/git.git` repo on win10

Reply:
Now there are a commit that uses `EnvUtils.ReplaceLinuxNewLinesDependingOnPlatform`